client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -0,0 +1,221 @@
+
+<template>
+  <div class="popover-switch">
+    <ion-title class="popover-switch-title body">{{ 'Connected organizations' }}</ion-title>
+    <ion-list class="orga-list">
+      <ion-item
+        class="orga-list__item body"
+        button
+        lines="none"
+        v-for="org in connectedOrgs"
+        :key="org.handle"
+        :disabled="org.active"
+        @click="onOrganizationClick(org)"
+      >
+        <div class="orga">
+          <ion-avatar class="orga-avatar">
+            <span>{{ org.id.substring(0, 2) }}</span>
+          </ion-avatar>
+          <div class="orga-text-content">
+            <ion-label class="orga-text-content__name">
+              <span class="body">
+                {{ org.id }}
+              </span>
+            </ion-label>
+            <ion-label class="orga-text-content__email">
+              <span class="body-sm">
+                {{ org.userLabel }}
+              </span>
+            </ion-label>
+          </div>
+
+          <ion-text
+            class="badge caption-caption"
+            v-show="org.active"
+            :outline="true"
+          >
+            {{ 'Active' }}
+          </ion-text>
+        </div>
+      </ion-item>
+
+      <ion-item
+        class="orga-list__item body"
+        button
+        lines="none"
+        @click="onAllOrganizationClick"
+      >
+        <div class="orga">
+          <ion-avatar class="orga-avatar ellipsis">
+            <ion-icon :icon="ellipsisHorizontal" />
+          </ion-avatar>
+          <div class="orga-text-content">
+            <ion-label class="orga-text-content__name">
+              <span class="body">
+                {{ 'Mes organisations' }}
+              </span>
+            </ion-label>
+          </div>
+        </div>
+      </ion-item>
+    </ion-list>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ConnectionHandle, OrganizationID, getLoggedInDevices } from '@/parsec';
+import { getConnectionHandle } from '@/router';
+import { IonAvatar, IonIcon, IonItem, IonLabel, IonList, IonText, IonTitle, popoverController } from '@ionic/vue';
+import { ellipsisHorizontal } from 'ionicons/icons';
+import { Ref, onMounted, ref } from 'vue';
+
+interface ConnectedOrganization {
+  id: OrganizationID;
+  userLabel: string;
+  active: boolean;
+  handle: ConnectionHandle;
+}
+
+const connectedOrgs: Ref<Array<ConnectedOrganization>> = ref([]);
+
+onMounted(async () => {
+  const result = await getLoggedInDevices();
+  connectedOrgs.value = result.map((info) => {
+    return {
+      id: info.device.organizationId,
+      active: getConnectionHandle() === info.handle,
+      handle: info.handle,
+      userLabel: info.device.humanHandle.label,
+    };
+  });
+});
+
+async function onOrganizationClick(org: ConnectedOrganization): Promise<void> {
+  if (org.handle === getConnectionHandle()) {
+    return;
+  }
+  await popoverController.dismiss({
+    handle: org.handle,
+  });
```

Comment:
I enjoy early exit conditions too, but in this case it does not simplify the reading, it's simpler to just invert the condition.
```suggestion
  if (org.handle !== getConnectionHandle()) {
    await popoverController.dismiss({
      handle: org.handle,
    });
  }
```